### DISCUSSION TITLE
Diferentes output en docker y en local

### DISCUSSION THREAD

**Discussion by antoniablanco** [IIC2133-PUC/2024-1#186] [Tarea 3] [answered]
Hola, tengo una duda. Un test especifico en local me entrega la respuesta correcta (556) pero al correrlo en el docker me entrega 24. Todos los otros test cases dan correcto. No veo como el docker puede estar cambiando la solución final, porque implicaría que esta cambiando el código o algo asi.

**Comment by benjavicente** [comment]
Hola! En que parte pasa ese problema?

Eso suele pasar cuando uno se topa con "comportamiento indefinido", es decir, algo que varia segun cimpilador/sistema operativo, como por ejemplo, la estabilidad de la función `qsort`.

Es extremadamente raro que afecte algo así en esta tarea. Danos más información como en que tests y parte es y te ayudamos a ver porque pasa y como arreglarlo.

**Comment by antoniablanco** [comment, marked as answer]
Al final, elimine y volví a crear el docker y ahora funciona. Me imagino que estaba pegado leyendo otro input.txt. Aunque no sé realmente cuál era el error.
Muchas gracias!

Obtener Outlook para iOS<https://aka.ms/o0ukef>
________________________________
De: Benjamín Vicente ***@***.***>
Enviado: Thursday, June 20, 2024 2:32:01 PM
Para: IIC2133-PUC/2024-1 ***@***.***>
Cc: Antonia Francisca Blanco Del Río ***@***.***>; Author ***@***.***>
Asunto: Re: [IIC2133-PUC/2024-1] Diferentes output en docker y en local (Discussion #186)


Hola! En que parte pasa ese problema?

Eso suele pasar cuando uno se topa con "comportamiento indefinido", es decir, algo que varia segun cimpilador/sistema operativo, como por ejemplo, la estabilidad de la función qsort.

Es extremadamente raro que afecte algo así en esta tarea. Danos más información como en que tests y parte es y te ayudamos a ver porque pasa y como arreglarlo.
